index.d.ts

```diff
@@ -120,32 +120,32 @@ declare namespace Eris {
     url?: string;
   }
   interface Activity extends ActivityPartial<ActivityType> {
+    // start undocumented props
     id: string;
+    sync_id?: string;
+    session_id?: string;
+    // end undocumented props
```

Comment:
Why do these need to be in the typings if they're undocumented?

Reply:
Good point, so far I've found id is incredibly useful when used in conjunction with created_at. When you start and stop you get a fresh id with the new timestamp. Can't speak much to the other ones.

Reply:
hm, what info do you get from `id` that you don't with `created_at`? I think in general we should probably avoid putting undocumented things in the typings.

Reply:
You could *probably* use created_at as the id, but there is technically a case where two presences are added at the same time having the same created_at. We can still remove them though and for this case I can just use some custom types.

Reply:
Removed in 8e1c4eb